go/variable_names.md

```diff
@@ -5,11 +5,24 @@ tend to use in our code. Aligning with the suggestions below makes our code
 easier to scan and code in general easier to write because it is a
 no-brainer how to call things, so we can concentrate on what actually matters.
 
-### Custom Resource
+## Controller Context
 
-In our `operatorkit` resource implementations, we take the parameter `obj`
-and have to assert the proper type the resource actually works with. The
-variable name we want to use here is `cr`.
+In `operatorkit` resource implementations, we sometimes work with
+`controllercontext` package. We store data extracted from this context in
+variable named `cctx`.
+
+```go
+cctx, err := controllercontext.FromContext(ctx)
```

Comment:
If `cctx` is the best name is debatable. Let's agree on something and start using it.

So far some call it:

- `cc` - I find it confusing with cloudconfig which we often assign to `cc` in our operator. Cloudconfig is a thing of the past though. So fine with me.
- `ctlCtx` - This is ok imo, but cumbersome to type.

So `cctx` is a mixture of both I just made up.

Reply:
My vote is for `ctlCtx`. It is longer but its clearer for me. `cctx` looks a bit like a typo to me.

Reply:
My vote goes on `cc`.

Reply:
I prefer `ctlCtx` as it avoid any colision in case we already have `cc` cloudconfig variable.

Reply:
When we don't like `cctx` and I looked at the code lately I prefer `cc`. We are even now. What should I do? :D

Reply:
When nobody vetos `cc` until tomorrow we go with it.

Reply:
I vote for `cc` let's get this over with.